test/driver/mysql_test.js

```diff
@@ -609,9 +609,9 @@ driver.connect(config, function(err, db) {
       }
     }
   }).addBatch({
-    'createMigrationsTable': {
+    '_createMigrationsTable': {
```

Comment:
I renamed it because i'm assuming the "_" signifies that it is supposed to be a private method and it should not be used by the users. Now that I think about it, In this case though it should probably not be private since it is used by migrator.js or migration.js.